02/Form.js

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
 
 class Form extends React.Component {
+	state = {
+		error: '',
+	};
```

Comment:
Tak jest ok, jeśli mamy stan, który dotyczy jednego komponentu i tej informacji nigdzie indziej nie potrzebujemy to taki stan lokalny jest jak najbardziej ok!